lib/widgets/app_bar/title_app_bar.dart

```diff
@@ -55,11 +55,18 @@ class TitleWidgetAppBar extends StatelessWidget implements PreferredSizeWidget {
       scrolledUnderElevation: 10,
       titleSpacing: 0,
       leadingWidth: 100,
-      title: title,
+      title: Container(
+        margin: const EdgeInsets.symmetric(
```

Comment:
I think this will collide with other usages of `TitleWidgetAppBar` where we want a text title. Please double check that every app bar with a text title still behaves as expected.